base/collect_database.go

```diff
@@ -57,8 +57,10 @@ type collectDatabaseInputs struct {
 }
 
 const (
-	// collectDatabaseTaskName is the name of this task which performs load generation and metrics collection for gRPC services.
+	// CollectDatabaseTaskName is the name of this task which performs load generation and metrics collection for gRPC services.
 	CollectDatabaseTaskName = "collect-metrics-database"
+	// experimentMetricsPathSuffix is the name of the metrics spec file
+	experimentMetricsPathSuffix = ".metrics.yaml"
```

Comment:
I see two constants ... "experimentMetricsPathSuffix" and "ExperimentMetricsPathSuffix"... combine?

Reply:
@sriumcp I forgot to point this out. If I were to use `ExperimentMetricsPathSuffix` here, I would need to bring in `driver` and it would give me an import cycle error. I was not sure what the best way to resolve this error is so I went with this solution for now.

Reply:
We will resolve this in a subsequent PR with changes too drivers.